kiwi/container/oci.py

```diff
@@ -207,6 +214,23 @@ def pack_image_to_file(self, filename):
             image_dir, xz_options=self.xz_options
         )
 
+    def _get_source(self):
+        if Defaults.is_buildservice_worker():
+            with open(os.sep + Defaults.get_buildservice_env_name()) as env:
+                for line in env:
+                    if line.startswith('BUILD_DISTURL') and '=' in line:
+                        return line.split('=')[1].strip()
+                log.warning('Could not find DISTURL inside .buildenv')
+
+        return None
```

Comment:
Returning None is not needed, that's the default

Reply:
Yes I know, I included it for clarity, but I can certainly live without it.